fuzzing/common.bzl

```diff
@@ -15,19 +15,22 @@
 
 """This file contains common rules for fuzzing test."""
 
+load("@bazel_skylib//rules:common_settings.bzl", "BuildSettingInfo")
+
 def _fuzzing_launcher_impl(ctx):
     # Generate a script to launcher the fuzzing test.
     script = ctx.actions.declare_file("%s" % ctx.label.name)
 
     script_template = """#!/bin/sh
-exec {launcher_path} {target_binary_path} --corpus_dir={corpus_dir} "$@"
-"""
+exec {launcher_path} {target_binary_path} --corpus_dir={corpus_dir} "$@" """
 
     script_content = script_template.format(
         launcher_path = ctx.executable._launcher.short_path,
         target_binary_path = ctx.executable.target.short_path,
         corpus_dir = ctx.file.corpus.short_path if ctx.attr.corpus else "",
     )
+    if ctx.attr._engine[BuildSettingInfo].value == "default":
+        script_content += " --regression=True"
```

Comment:
I'm not sure this is right: The engine and whether it runs in regression mode or not should be orthogonal. We should also be able to run libFuzzer in regression mode, for example.

Reply:
Yep, right now it looks like `--config asan-libfuzzer` dictates whether it runs in regression mode or not, but the command name dictates whether it is in regression mode:
`bazel test <name>` vs `bazel run <name>_run`